website/pages/en/community.js

```diff
@@ -0,0 +1,73 @@
+/**
```

Comment:
There's already "Community" page -> https://facebook.github.io/react-native/help (the slug is confusing, it's named Community from the top bar). Maybe we could use that one somehow?

Reply:
I put this in the PR description:

> There is currently a "community" link in the header, however, this goes to the "help" page which then includes help links and links to 3rd-party communities. This is useful information so we should keep it.
>
> To me, it makes sense to have the new Community page go to the new page in this PR and have the current Help page accessible under a new "Help" link in the header.

I don't really mind though and I'm happy to go with whatever people think on the website structure.